lizmap/modules/lizmap/controllers/edition.classic.php

```diff
@@ -364,6 +364,7 @@ private function getDataFields($datasource){
     // Set some private properties
     $this->table = $table;
     $this->tableName = $tableAlone;
+    $this->schemaName = $schema;
```

Comment:
This changes is not needed, there is already `$this->schema = $chema`

Reply:
I was too tired yesterday.